File: Klijent/Film.js
export class Film{

    constructor(sifra, naziv, zanr, filmoteka){
        this.sifra=sifra;
        this.naziv=naziv;
        this.zanr=zanr;
        this.filmoteka=filmoteka;
    }

    crtaj(host){
       
        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML=this.sifra;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.naziv;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.zanr;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML=this.filmoteka;
        tr.appendChild(el);
    }
}

File: Models/Film.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Models
{
    [Table("Film")]
    public class Film
    {
        [Key]
        public int ID { get; set; }


        [Required]
        public int Sifra { get; set; }


        [Required]
        [MaxLength(50)]
        public string Naziv { get; set; }

        public virtual Filmoteka Filmoteka{ get; set; }

        public virtual Zanr Zanr{ get; set; }
        
        [JsonIgnore]
        public virtual List<Iznajmljivanje> FilmKorisnik { get; set; }

    }


}